src/luascript.cpp

```diff
@@ -11550,6 +11551,18 @@ int32_t LuaScriptInterface::luaItemTypeGetDecayId(lua_State* L)
 	return 1;
 }
 
+int32_t LuaScriptInterface::luaItemTypeGetRequiredLevel(lua_State* L)
+{
+	// itemType:getRequireLevel()
+	const ItemType* itemType = getUserdata<const ItemType>(L, 1);
+	if (itemType) {
+		lua_pushnumber(L, itemType->minReqLevel);
+	}else {
```

Comment:
@Codinablack 
Don't take it so personally.
I saw a tiny typo you had made and commented it out, not trying bring you down or anything, rather trying to help you.

I feel you with the git issues though, if I had noticed you were having them, I probably wouldn't have commented anything.